shortcut/__init__.py

```diff
@@ -1,3 +1,4 @@
+from .exception import *
```

Comment:
Why such a broad import ? Can you import the specific exceptions?

Reply:
Am, It wasn't me, it was you to write in this way :)